hbase_sql/src/main/java/com/splicemachine/olap/OlapServer.java

```diff
@@ -56,7 +56,9 @@ public void startServer(SConfiguration config) {
         ServerBootstrap bootstrap = new ServerBootstrap(factory);
 
         // Instantiate handler once and share it
-        OlapJobRegistry registry = new MappedJobRegistry(1L,TimeUnit.SECONDS); //TODO -sf- make this tick time configurable
+        OlapJobRegistry registry = new MappedJobRegistry(config.getOlapClientTickTime(),
+                config.getOlapServerTickLimit(),
+                TimeUnit.MILLISECONDS);
```

Comment:
Changed from seconds to milliseconds but default time stayed the same.  Is that what you wanted?